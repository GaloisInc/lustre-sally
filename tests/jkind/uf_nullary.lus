-- This file exercises nullary functions
-- Expected output:
-- VALID PROPERTIES: [ok]
-- INVALID PROPERTIES: [cex] (k = 1)

function f() returns (x, y: int);
function g(x, y : int) returns ();

node main() returns (ok, cex: bool);
var
  x, y : int;
let
  ok = (g(1, 2) = g(3, 4));
  --%PROPERTY ok;

  x, y = f();
  cex = not (x = 5);
  --%PROPERTY cex;
tel;
