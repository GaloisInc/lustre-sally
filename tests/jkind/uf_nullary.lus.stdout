[Lustre] Loading model from: "uf_nullary.lus"
[Error] Parse error at 7:10

